RemoveOutroMod/RemoveOutroMod.lua

```diff
@@ -0,0 +1,10 @@
+ModUtil.RegisterMod("RemoveOutroMod")
+
+ModUtil.BaseOverride("EndEarlyAccessPresentation", function ()
+	CurrentRun.ActiveBiomeTimer = false
+	
+	thread( Kill, CurrentRun.Hero )
+	wait( 0.15 )
+
+	FadeIn({ Duration = 0.5 })
+end, RemoveOutroMod)
```

Comment:
github complains that you didn't end with a newline.  Probably worth adding even if its unlikely to affect anything we will do with this file